low-level/framework/base/sspl_constants.py

```diff
@@ -36,6 +36,7 @@
 RESOURCE_PATH = "/opt/seagate/%s/sspl/low-level/json_msgs/schemas/" % (PRODUCT_FAMILY)
 CLI_RESOURCE_PATH = "/opt/seagate/%s/sspl/low-level/tests/manual" % (PRODUCT_FAMILY)
 DATA_PATH = "/var/%s/sspl/data/" % (PRODUCT_FAMILY)
+IEM_DATA_PATH = "/var/%s/sspl/data/iem/sspl_iem"  %(PRODUCT_FAMILY)
```

Comment:
"/var/%s/sspl/data/iem/" folder is enough as its under sspl already !?

Reply:
"/var/%s/sspl/data/iem/" this will also contain last_processed_msg_time file, hence inside this data path creates one more dir ie. sspl_iems and under that keeping event files for each sspl iem_event, 
for eg. iem_IPMITOOL, iem_HDPARM, iem_KAFKA.. etc. 
is it fine or should I remove sspl_iems dir??

Reply:
thats fine